### PR TITLE
Adds support for slice65 from pizzakeyboards

### PR DESCRIPTION
**Pull request by cskeeters**
## Description

Adds support for slice65 from pizzakeyboards

## QMK Pull Request 

The following pull request was merged into master yesterday.
https://github.com/qmk/qmk_firmware/pull/23148

## Checklist

- [x] The VIA support for this keyboard is **MERGED** in QMK master already **(MANDATORY)**
- [x] The VIA definition follows the guide here: https://caniusevia.com/docs/layouts
- [x] I have a V3 JSON version for this keyboard definition.**(MANDATORY)**
- [x] I have tested this keyboard definition using VIA's "Design" tab.
- [x] I have tested this keyboard definition with firmware on a device.
- [x] I have assigned alpha keys and modifier keys with the correct colors.
- [x] The Vendor ID is not `0xFEED`

### CODE REVIEW

**Comment by Cipulot** [comment]
@cskeeters sorry for being pedantic, but could you revise the KLE key colors to match the specs in the documentation?
As of now, this is the KLE situation and the rendered VIA view.
Thanks 😄 
image
image

**Comment by cskeeters** [comment]
I put in the key colors according to <https://www.caniusevia.com/docs/layouts>.  In the default OLIVIA DARK theme, keys with color #cccccc and #777777 both render black.  When you switch to the theme named OLIVE, the accents show as a unique color.  If there is something I am missing, or can do differently, just let me know.  I may be misunderstanding something.

**Comment by Cipulot** [comment]
> I put in the key colors according to https://www.caniusevia.com/docs/layouts. In the default OLIVIA DARK theme, keys with color #cccccc and #777777 both render black. When you switch to the theme named OLIVE, the accents show as a unique color. If there is something I am missing, or can do differently, just let me know. I may be misunderstanding something.

Even switching to another theme, there's is a mix of the `cccccc` and `777777` in mods and accents. Some mods are marked as accent keys and others are not.
Screenshot 2024-03-27 at 4 15 38 PM

The note in the previous comment was about that, rather than the fact that mods and alphas had the same color in the particular theme.

**Comment by cskeeters** [comment]
### Modifiers

I wonder if we have a different definition of a Modifier Key (mods).  [Wikipedia](https://en.wikipedia.org/wiki/Modifier_key) defines a modifier key as:

> In computing, a modifier key is a special key (or combination) on a computer keyboard that temporarily modifies the normal action of another key when pressed together.

To me, this means, <kbd>Shift</kbd>, <kbd>Ctrl</kbd>, <kbd>Alt</kbd>, and arguably <kbd>Gui</kbd>.  I believe I have colored all and only the modifier keys #aaaaaa.  Can you give me an example of a modifier that is colored as an accent?

### Accents

Honestly, I don't know what key counts as an accent.  To me, the accent of a keyboard has more to do with the coloring of the keycaps which doesn't make sense on a keyboard that comes with no keycaps.  On the page <https://www.caniusevia.com/docs/layouts>, it says: 

> Use key color #cccccc for alphas, #aaaaaa for modifier keys and #777777 for accents (i.e. Esc, Enter, arrows, etc).

Since it lists Esc, Enter, and arrows as accents, I decided to color any key that was not an alpha and not a modifier as an accent.  If you can list the keys that are accents, or describe how I'm supposed to decide which keys to consider accents, I'm open to that feedback.

**Comment by Cipulot** [comment]
> Can you give me an example of a modifier that is colored as an accent?
Screenshot 2024-03-29 at 3 57 52 PM

The highlighted keys don't match the criteria for them to be accent colored, special notes are given to Tab, Caps, and Backspace. The function keys in this case configured best as #aaaaaa colored keys. The backslash symbol was marked as an accent for seemingly no reason.

I'll move on and align the colors with the average usage in the rest of the repo as follows:
Screenshot 2024-03-29 at 3 59 41 PM
As well as apply code formatting to remove empty lines and align the space usage.

File: src/pizzakeyboards/slice65/slice65.json [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+                    "x": 0.5,
+                    "c": "#777777"
+                },
+                "8,6",
+                "9,6",
+                "8,7"
+            ]
+        ]
+    }
+}
```
Comment (Cipulot) [inline src/pizzakeyboards/slice65/slice65.json:163]:
```suggestion
{
  "name": "slice65",
  "vendorId": "0x504b",
  "productId": "0x707c",
  "matrix": {
    "rows": 10,
    "cols": 8
  },
  "layouts": {
    "keymap": [
      [
        {
          "c": "#777777"
        },
        "0,0",
        {
          "c": "#cccccc"
        },
        "1,0",
        "0,1",
        "1,1",
        "0,2",
        "1,2",
        "0,3",
        "1,3",
        "0,4",
        "1,4",
        "0,5",
        "1,5",
        "0,6",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "1,6",
        "0,7"
      ],
      [
        {
          "w": 1.5
        },
        "2,0",
        {
          "c": "#cccccc"
        },
        "3,0",
        "2,1",
        "3,1",
        "2,2",
        "3,2",
        "2,3",
        "3,3",
        "2,4",
        "3,4",
        "2,5",
        "3,5",
        "2,6",
        {
          "w": 1.5
        },
        "3,6",
        {
          "c": "#aaaaaa"
        },
        "2,7"
      ],
      [
        {
          "w": 1.75
        },
        "4,0",
        {
          "c": "#cccccc"
        },
        "5,0",
        "4,1",
        "5,1",
        "4,2",
        "5,2",
        "4,3",
        "5,3",
        "4,4",
        "5,4",
        "4,5",
        "5,5",
        {
          "c": "#777777",
          "w": 2.25
        },
        "5,6",
        {
          "c": "#aaaaaa"
        },
        "4,7"
      ],
      [
        {
          "w": 2.25
        },
        "6,0",
        {
          "c": "#cccccc"
        },
        "6,1",
        "7,1",
        "6,2",
        "7,2",
        "6,3",
        "7,3",
        "6,4",
        "7,4",
        "6,5",
        "7,5",
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "6,6",
        {
          "c": "#777777"
        },
        "7,6",
        {
          "c": "#aaaaaa"
        },
        "6,7"
      ],
      [
        {
          "w": 1.25
        },
        "8,0",
        {
          "w": 1.25
        },
        "9,0",
        {
          "w": 1.25
        },
        "8,1",
        {
          "c": "#cccccc",
          "w": 6.25
        },
        "8,3",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "8,5",
        {
          "w": 1.25
        },
        "9,5",
        {
          "x": 0.5,
          "c": "#777777"
        },
        "8,6",
        "9,6",
        "8,7"
      ]
    ]
  }
}
```

**Review by Cipulot — CHANGES_REQUESTED** [review]

**Review by Cipulot — APPROVED** [review]

**Comment by Cipulot** [comment]
@cskeeters I have applied the changes to the layout. Pull the latest changes and move the file form `src/pizzakeyboards/slice65/slice65.json` to `v3/pizzakeyboards/slice65/slice65.json`, as per requirement n.3 in the PR Checklist.

**Pull request merged**